cordova-js-src/platform.js

```diff
@@ -58,6 +61,19 @@ module.exports = {
         bindButtonChannel('volumeup');
         bindButtonChannel('volumedown');
 
+        // The resume event is not "sticky", but it is possible that the event
+        // will contain the result of a plugin call. We need to ensure that the
+        // plugin result is delivered even after the event is fired (CB-10498)
+        var cordovaAddEventListener = document.addEventListener;
```

Comment:
@adamduren We already patch `addEventListener` in the common Cordova js code (hence the `cordovaAddEventListener` name). See https://github.com/apache/cordova-js/blob/master/src/cordova_b.js#L48

Come to think of it, I should probably be doing the method in window as well. I'll update the PR. Thanks for taking a look!